pkg/prometheus.go

```diff
@@ -9,6 +9,48 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	cloudwatchAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	cloudwatchAPIErrorCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_request_errors",
+		Help: "Help is not implemented yet.",
+	})
+	cloudwatchGetMetricDataAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_getmetricdata_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	cloudwatchGetMetricStatisticsAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_getmetricstatistics_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	resourceGroupTaggingAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_resourcegrouptaggingapi_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	autoScalingAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_autoscalingapi_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	targetGroupsAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_targetgroupapi_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	apiGatewayAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_apigatewayapi_requests_total",
+	})
+	ec2APICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_ec2api_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+	dmsAPICounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "yace_cloudwatch_dmsapi_requests_total",
+		Help: "Help is not implemented yet.",
+	})
+)
```

Comment:
Should `Metrics` slice be declared next to the metrics now

Reply:
My thought is that `update.go` is the library entrypoint so if we centralize important pieces of the library API it can make it more discoverable/obvious. We might be the only people using it as a library but might help upstream maintenance as a reminder there's multiple entrypoints. Writing this made think it might be a good idea to add this info to the upstream README. WDYT?

Reply:
> We might be the only people using it as a library but might help upstream maintenance as a reminder there's multiple entrypoints. Writing this made think it might be a good idea to add this info to the upstream README. WDYT?

+1